WordPress/src/main/java/org/wordpress/android/ui/stats/StatsActivity.java

```diff
@@ -143,7 +150,7 @@ public void onRefreshStarted() {
                         mRequestedDate = StatsUtils.getCurrentDateTZ(mLocalBlogID);
                         loadStatsFragments(false, true, true); // This is here just for a security check
```

Comment:
Just a suggestion for the future: routines like this which have several parameters of the same type (boolean in this case) can be error-prone for future devs since they're not self-documenting and it's easy to get the order of parameters mixed up. Using a single EnumSet parameter is cleaner, ex:
```
static enum LoadStatsOption {
    FORCE_RECREATION,
    LOAD_GRAPH_FRAGMENT,
    LOAD_ALLTIME_FRAGMENTS
}
private void loadStatsFragments(EnumSet<LoadStatsOption> options) {
    ...
}
```